wdio.conf.js

```diff
@@ -6,6 +6,21 @@ const wdioConfig = defaultWdioConfig.config;
 
 if (process.env.npm_package_name !== 'terra-framework') {
   wdioConfig.specs = [path.join(__dirname, 'packages', process.env.npm_package_name, 'tests', 'wdio', '**', '*-spec.js')];
+} else {
+  wdioConfig.specs = [
+    path.join(__dirname, 'packages', 'terra-abstract-modal', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-aggregator', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-application-header-layout', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-application-layout', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-application-links', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-application-menu-layout', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-application-name', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-application-navigation', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-application-utility', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-brand-footer', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-collapsible-menu-view', 'tests', 'wdio', '**', '*-spec.js'),
+    path.join(__dirname, 'packages', 'terra-date-input', 'tests', 'wdio', '**', '*-spec.js'),
+  ];
```

Comment:
This is to prevent wdio tests in all the other packages from running. This will be reverted in the [last PR (Part 3/3)](https://github.com/cerner/terra-framework/pull/1368/files#diff-f297ad33212595210fbfc7c32d334f7025fe8eee14cc4179d2527c3f4543035f).